eng/common/performance/performance-setup.sh

```diff
@@ -2,20 +2,23 @@
 
 source_directory=$BUILD_SOURCESDIRECTORY
 core_root_directory=
+baseline_core_root_directory=
 architecture=x64
-framework=netcoreapp3.0
+framework=netcoreapp5.0
```

Comment:
netcoreapp3.1?

Reply:
These are in eng/common. Should be left as-is.